map_frame: map
base_frame: base_footprint
thresh: 50
clear_frontiers: 40